Doc: [WD-18945] Add docs for create a network (#15391)
canonical/lxd: doc/howto/network_create.md

```

Also see {ref}`cluster-config-networks`.
````
```` {group-tab} UI

From the main navigation, select {guilabel}`Networks`.

On the resulting page, click {guilabel}`Create network` in the upper-right corner.

> You can then configure the network name and type, as well as other attributes. Optional additional attributes are split into the categories {guilabel}`Bridge`, {guilabel}`IPV4`, {guilabel}`IPV6` and {guilabel}`DNS`, which can be seen in the sub-menu on the right side of the screen.

Reviewer: IPv4 and IPv6 please

> You can then configure the network name and type, as well as other attributes. Optional additional attributes are split into the categories {guilabel}`Bridge`, {guilabel}`IPv4`, {guilabel}`IPv6` and {guilabel}`DNS`, which can be seen in the sub-menu on the right side of the screen.

Author: "submenu" fails the spelling check @minaelee :). Is there any way I can circumvent this? Or is "sub-menu" okay?

> You can then configure the network name and type, as well as other attributes. Optional additional attributes are split into the categories {guilabel}`Bridge`, {guilabel}`IPv4`, {guilabel}`IPv6` and {guilabel}`DNS`, which can be seen in the sub-menu on the right side of the screen.

Reviewer 2: > "submenu" fails the spelling check @minaelee :). Is there any way I can circumvent this? Or is "sub-menu" okay?

Let's circumvent it. Submenu is the primary accepted spelling in all three major dictionaries I checked, as well as [this UX glossary](https://www.nngroup.com/articles/ui-elements-glossary/#Submenu).

There are two ways to circumvent spellchecks. One is on a case-by-case basis only for words you don't always want to ignore during spellcheck, via the `{spellexception}` directive. The other way, which we should use here, is to add it to a site-wide ignore list. You'll find that in `doc/.custom_wordlist.txt`. Please only add words to this list that should always be ignored. TIA!

    lxc network attach my-network my-instance eth0

````
```` {group-tab} UI

> When creating or configuring an instance, from the {guilabel}`Devices` section in the left hand sub-menu, select the {guilabel}`Network` tab to view and edit networks linked to the instance.

Reviewer 2: ```suggestion
When {ref}`creating <instances-create>` or {ref}`configuring an instance <instances-configure>`, go to the {guilabel}`Devices` section in the left-hand submenu, then select {guilabel}`Network` to view and edit the networks linked to the instance.
```

```{figure} /images/networks/network_add_to_instance.png
:width: 80%
:alt: Add a network to an instance in LXD
```

> Click the {guilabel}`Attach network` button within the page to add a new network. From here you can select a previously created network from the {guilabel}`Network` dropdown, and assign it a device name.

Reviewer 2: ```suggestion
Click the {guilabel}`Attach network` button to add a new network. From here, you can select an existing network from the {guilabel}`Network` dropdown and assign it a device name.
```
